### ISSUE TITLE
[Accessibility]: The dropdown (code implementation) is not accessible with keyboard

### ISSUE THREAD

**Issue by afrahB** [Orange-OpenSource/ods-android#750]
### Prerequisites

- [X] I have [searched the backlog](https://github.com/Orange-OpenSource/ods-android/issues) for duplicate or closed feature requests

### Your test device

Samsung A53

### OS Version

Android 14

### App version

1.0

### Describe the issue

impossible to navigate to dropdown (code implementation) with a keyboard 
the focus is not visible
unable to switch the implementation's code (xml/jetpack)

### Expected Behavior

the dropdown is focusable when it's activated 
the user can swith between implementations
the focus does not remain blocked in the menu
